### ISSUE TITLE
Run make image with error in "Configuration API demo with Etcd"

### ISSUE THREAD

**Issue by shenjing023** [mosn/layotto#439]
**What happened**:
In Configuration API demo with Etcd step, run `make image` and error happened
```
docker build --rm -t godep-builder build/contrib/builder/binary
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM golang:1.16
 ---> 8ffb179c0658
Step 2/2 : MAINTAINER wangwenxue.wwx@antgroup.com
 ---> Running in 0b35d9e28e63
Removing intermediate container 0b35d9e28e63
 ---> 70e66ae2a999
Successfully built 70e66ae2a999
Successfully tagged godep-builder:latest
docker run --rm -v /home/liuwei1/layotto:/go/src/mosn.io/layotto -w /go/src/mosn.io/layotto godep-builder make build-local
CGO_ENABLED=1 go build \
        -ldflags "-B 0x0fc65211d185d42b91ade91fb49185bc4415c64f -X main.Version=0.2.0(428540f) -X mosn.io/layotto/pkg/types.IstioVersion=" \
        -v -o layotto \
        mosn.io/layotto/cmd/layotto
go: github.com/99designs/keyring@v1.2.1: Get "https://proxy.golang.org/github.com/99designs/keyring/@v/v1.2.1.mod": dial tcp 172.217.160.113:443: i/o timeout
make: *** [Makefile:18: build-local] Error 1
make: *** [image] Error 2
```
**What you expected to happen**:
No error
**How to reproduce it (as minimally and precisely as possible)**:
Just run `make image`
**Anything else we need to know?**:
For the Chinese network, add GOPROXY in the Makefile
```
build-local:
	@rm -rf build/bundles/${MAJOR_VERSION}/binary
	go env -w GOPROXY=https://goproxy.cn,direct
	CGO_ENABLED=1 go build \
```

**Comment by seeflood** [comment]
Thanks for your feedback !
Could you modify your local GOPROXY permanently by running `go env -w GOPROXY=https://goproxy.cn,direct` ?
If we add GOPROXY in the Makefile ,the github workflow will use chinese proxy too,which might also lead to timeout ....

**Comment by shenjing023** [comment]
I had set go env, but also failed
```
[liuwei1@centos layotto]$ go env
GO111MODULE="on"
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/liuwei1/.cache/go-build"
GOENV="/home/liuwei1/.config/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/liuwei1/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/liuwei1/go"
GOPRIVATE=""
GOPROXY="https://goproxy.cn,direct"
GOROOT="/data/home/liuwei1/go1.18/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/data/home/liuwei1/go1.18/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.18"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/liuwei1/layotto/go.mod"
GOWORK=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build1726670767=/tmp/go-build -gno-record-gcc-switches"
[liuwei1@centos  layotto]$ make image
docker build --rm -t godep-builder build/contrib/builder/binary
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM golang:1.16
 ---> 8ffb179c0658
Step 2/2 : MAINTAINER wangwenxue.wwx@antgroup.com
 ---> Using cache
 ---> 86c4f2e797de
Successfully built 86c4f2e797de
Successfully tagged godep-builder:latest
docker run --rm -v /home/liuwei1/layotto:/go/src/mosn.io/layotto -w /go/src/mosn.io/layotto godep-builder make build-local
CGO_ENABLED=1 go build \
        -ldflags "-B 0xe930dff69242cf06fcf2214378509210ebb9c671 -X main.Version=0.2.0(428540f) -X mosn.io/layotto/pkg/types.IstioVersion=" \
        -v -o layotto \
        mosn.io/layotto/cmd/layotto
go: github.com/99designs/keyring@v1.2.1: Get "https://proxy.golang.org/github.com/99designs/keyring/@v/v1.2.1.mod": dial tcp 172.217.163.49:443: i/o timeout
make: *** [Makefile:18: build-local] Error 1
```

**Comment by seeflood** [comment]
You are right. I can reproduce this issue at home now 😿 
It's because the `docker run` command build the layotto in a container without GOPROXY.
I will fix it.

**Comment by seeflood** [comment]
It wil be fixed in #441
